AdofaiTweaks/Tweaks/HideUiElements/HideUiElementsSettings.cs

```diff
@@ -71,12 +72,17 @@ public class HideUiElementsSettings : TweakSettings
 
             bool hideEverything = selectedProfile.HideEverything;
             bool hideOtto = hideEverything || selectedProfile.HideOtto;
+            bool hideTimingTarget = hideEverything || selectedProfile.HideTimingTarget;
+            bool hideNoFail = hideEverything || selectedProfile.HideNoFailIcon;
             bool hideBeta = hideEverything || selectedProfile.HideBeta;
             bool hideTitle = hideEverything || selectedProfile.HideTitle;
 
             var tweakEnabled = AdofaiTweaks.IsEnabled && IsEnabled;
             hideEverything &= tweakEnabled;
             hideOtto &= tweakEnabled;
+            hideTimingTarget &= tweakEnabled;
+            hideNoFail &= tweakEnabled;
+            hideTitle &= tweakEnabled;
```

Comment:
`hideTitle` is already set later in line 87, so you can delete this.

Reply:
My bad, I accidentally duplicated it when adding the 2 new options. I'm removing it